components/auth/org.wso2.carbon.auth.token.introspection/pom.xml

```diff
@@ -45,4 +46,24 @@
             <artifactId>oauth2-oidc-sdk</artifactId>
         </dependency>
     </dependencies>
+
+    <properties>
+        <private.package>org.wso2.carbon.auth.core.internal</private.package>
```

Comment:
This needs to be removed otherwise the package will be copied to the bundle errornously